lib/editing/editor.dart

```diff
@@ -38,6 +39,8 @@ abstract class Editor {
   String get theme;
   set theme(String str);
 
+  Point get cursorCoords;
```

Comment:
Can you add some dartdoc to the effect that this may not be supported on all editors?